doc/classes/ItemList.xml

```diff
@@ -479,6 +482,9 @@
 		<theme_item name="focus" data_type="style" type="StyleBox">
 			The focused style for the [ItemList], drawn on top of the background, but below everything else.
 		</theme_item>
+		<theme_item name="hovered" data_type="style" type="StyleBox">
+			[StyleBox] for the hovered, not yet selected items.
```

Comment:
Why is this description different from `font_hovered_color`, if their logic is the same?

Reply:
Ajusted, the logic is the same